apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-start-node/ba-decision-graph-start-node.scss

```
@@ -0,0 +1,70 @@
+@import '../../../styles/global/variables';
+@import 'colors';
+
+:host {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+:host ::ng-deep button {
+  font-weight: normal;
+}
+
+:host ::ng-deep .ba-uxd-startnode p {
+  margin-bottom: 20px;
+}
+
+.ba-uxd-startnode-grid {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  width: 100%;
+  height: 100%;
+  margin: 52px 12px 40px;
+}
+
+@media only screen and (min-width: $screen-md) {
+  .ba-uxd-startnode-grid {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media only screen and (min-width: $screen-lg) {
+  .ba-uxd-startnode-grid {
+    grid-template-columns: repeat(2, 1fr);
+    width: 76%;
+    margin: 52px auto 40px;
+  }
+}
+
+.ba-uxd-startnode {
+  width: auto;
+  background-color: $royalblue-800;
+  border: 1px solid $gray-300;
+  border-radius: 4px;
+  margin-right: 8px;
+  margin-bottom: 8px;
+  padding: 0;
+  outline: 0;
+
+  &:hover {
+    background-color: $royalblue-600;
+    cursor: pointer;
+  }
+
+  &:active {
+    background-color: $royalblue-600;
+  }
```

Can be grouped to avoid duplication.
```suggestion
  &:hover,
  &:active {
    background-color: $royalblue-600;
  }
```